portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,11 +37,26 @@ public void init(){
         quotes.add("You can't have everything. Where would you put it?");
         quotes.add("Well-behaved women seldom make history.");
     }
+    public ArrayList messages(){
+        m = new ArrayList<>();
+        m.add("I am a message.");
+        m.add("Wow that last message is so self aware.");
+        m.add("Wait, you all can talk?");
+        return m;
+    }
+    public String toJson(ArrayList<String> alist){
+        Gson gson = new Gson();
+        String json = gson.toJson(alist);
+        return json;
+    }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String quote = quotes.get((int) (Math.random() * quotes.size()));
-    response.setContentType("text/html;");
-    response.getWriter().println(quote);
+    ArrayList<String> msg = messages();
+    String message = toJson(msg);
```

Comment:
Consider renaming from `message` to something like `jsonString` or `messageJsonString` (because `message` is used in a lot of places to mean a single message).

Reply:
Done